markymark/Classes/Protocols/CanSetURLOpener.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  CanSetURLOpener.swift
+//  Created by Jim van Zummeren on 05/07/2021.
+//
+
+import Foundation
+
+protocol CanSetURLOpener {
```

Comment:
Why did you decide to go for a setter function-based protocol over having a property-based one that allows setting?

Reply:
Discussed internally, it's decided to go for a setter through a function only because a getter is not needed.